installer/dev/install.cpp

```diff
@@ -289,6 +300,40 @@ namespace WindowsAppRuntimeInstaller
     }
     CATCH_RETURN()
 
+    // RestartPushNotificationsLRP is best effort and non-blocking to Installer functionality
+    void RestartPushNotificationsLRP()
```

Comment:
Probably needs to be a helper method somewhere? You ssem to be initializing LRP from several other components in your code base.

Reply:
LRP restart code is slightly different for Installer vs the other two locations because Installer is not built in the same WAR build. More details explained in other comments.

Even otherwise, there is quite a bit of overlap of code between components and inherited it from 1.0. After 1.1 code complete, I plan to pull out common parts for easy code maintenance over time. For this PR, this is it.